composer.json

```diff
@@ -126,7 +126,7 @@
         "drupal/views_bulk_operations": "^3.6",
         "drupal/xmlsitemap": "~1.0",
         "ext-imagick": "*",
-        "su-sws/jumpstart_ui": "dev-8.x-1.x",
+        "su-sws/jumpstart_ui": "dev-D8CORE-2376 as dev-8.x-1.x",
```

Comment:
apparently this only works at the main level composer.json. I had to add it to the stack to test.

Reply:
Yeah, it is root only.

Reply:
Much better than the alternative, however.